articles/subject-behavior.md

```diff
@@ -0,0 +1,13 @@
+---
+uid: subject-behavior
+title: "BehaviorSubject"
+---
+
+![Marble diagram](~/images/language-subject-behavior.svg)
+
+`BehaviorSubject` stores and passes the latest value emitted by the source sequence to each subscribed observer, and then continues to emit any subsequent values.
+
+Any observers which subscribe later will immediately receive the latest stored value. However, if the source sequence terminates with an error, `BehaviorSubject` will not emit any values but will pass along the error notification to all subsequent observers.
```

Comment:
Since it is such a prevalent motif, maybe use the "SelectMany" as an example ?

Reply:
We should list a series of examples for each of these subjects (and other operators), but since the SelectMany is not easy to explain by itself, I will leave it for a later PR (where we can also try to start cross-linking examples).